cpp/models/abm/parameters.h

```diff
@@ -230,13 +231,13 @@ struct MaximumContacts {
 using LocalInfectionParameters = ParameterSet<MaximumContacts>;
 
 struct TestParameters {
-    double sensitivity;
-    double specificity;
+    UncertainValue sensitivity;
+    UncertainValue specificity;
 };
 
 struct GenericTest {
     using Type = TestParameters;
-    static constexpr Type get_default()
+    static Type get_default()
```

Comment:
It was because constexpr cannot accept non-literal type. I switch them back since I think both sensitivity and specificity of the test can be specified at the beginning of the simulation and they don't necessary have to be UncertainValue.

Reply:
I see, yes. However, I think there might be some uncertainty in these values, so I would leave them as UncertainValues and remove the constexpr as you did before. Sorry for the back and forth.

Reply:
No problem. It's fixed!